client-src/elements/chromedash-activity-log.ts

```diff
@@ -72,11 +72,13 @@ export class ChromedashAmendment extends LitElement {
 customElements.define('chromedash-amendment', ChromedashAmendment);
 
 export interface User {
+  // TODO(markxiong0122): Move this interface to chromedash-app.js after migration.
   can_create_feature: boolean;
   can_edit_all: boolean;
   can_comment: boolean;
   is_admin: boolean;
   email: string;
+  approvable_gate_types?: any;
```

Comment:
The type itself is initialized in https://github.com/GoogleChrome/chromium-dashboard/blob/b0049ed27615bb01a4063924557bf8a0f5f36806/api/permissions_api.py#L45-L65
which implies that
```suggestion
  approvable_gate_types: int[];
  editable_features: int[];
```
finishes the type.